docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -173,12 +173,29 @@ for customer data including session recordings and user records.
 
 ## Can I get a list of IP addresses to configure a firewall?
 
-We don't currently offer a list of addresses for firewall configuration and plan to make this available in the future.
+We do not have plans to offer a list of IP addresses for Teleport Enterprise
+Cloud at the moment.
 
-## Can I configure Teleport Enterprise Cloud to only allow connections from an allowlist of IP addresses?
+Teleport Enterprise Cloud infrastructure is elastic and IP addresses can and do
+change when we make infrastructure changes. This would make the maintenance of
+IP allowlists a cumbersome and brittle process for customers.
 
-No. We believe mTLS with strong identity is the best standard available for authenticating and access clients.
-We are also investigating additional and new features that can be used as a defense in depth strategy for use with Teleport Enterprise Cloud in 2022.
+However, we do provide customers with a stable tenant DNS name backed with a
+TLS certificates from [letsencrypt.org](https://letsencrypt.org) that can be
```

Comment:
I know this is already merged, but you're mixing singular and plural with "backed with a TLS certificates".